src/lib/Scenes/Show/index.tsx

```diff
@@ -63,32 +64,46 @@ export class Show extends React.Component<Props, State> {
     /* TODO: implement */
   }
 
+  renderItemSeparator = () => (
+    <Box py={2} px={2}>
+      <Separator />
+    </Box>
+  )
+
+  renderItem = ({ item: { data, type } }) => {
+    switch (type) {
+      case "location":
+        return <Location show={data} />
+      case "artworks":
+        return <Artworks show={data} />
+      case "artists":
+        return <Artists show={data} />
+      case "shows":
+        return <Shows shows={data} />
+      default:
+        return null
+    }
+  }
+
   render() {
     const { show } = this.props
     return (
       <Theme>
         <FlatList
           data={this.state.sections}
           ListHeaderComponent={
-            <ShowHeader
-              show={show}
-              onSaveShowPressed={this.handleSaveShow}
-              onMoreInformationPressed={this.handleMoreInformationPressed}
-            />
+            <>
+              <ShowHeader
+                show={show}
+                onSaveShowPressed={this.handleSaveShow}
+                onMoreInformationPressed={this.handleMoreInformationPressed}
+              />
+              {this.renderItemSeparator()}
+            </>
           }
-          renderItem={({ item: { data, type } }) => {
-            switch (type) {
-              case "location":
-                return <Location show={data} />
-              case "artworks":
-                return <Artworks show={data} />
-              case "artists":
-                return <Artists artists={data} />
-              case "shows":
-                return <Shows shows={data} />
-              default:
-                return null
-            }
+          ItemSeparatorComponent={this.renderItemSeparator}
+          renderItem={data => {
+            return <Box px={2}>{this.renderItem(data)}</Box>
```

Comment:
Our separator and container styling is consistent so far, so it makes sense to lift it up into the `FlatList`.